app/views/admin/playlist_activities/_form.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= simple_form_for([:admin, @student, resource], html: { class: 'inline' }) do |form| %>
+  <%= render 'error_notification', form: form %>
+  <%= form.association :activity, collection: @student.grove.activities %>
+  <%= form.association :focus_area,
+                       collection: @student.grove.focus_areas,
```

Comment:
lets try to keep secret ActiveRecord queries out of the views. put this `@student.grove.focus_areas` into a helper method or instance variable in the controller. you can optimize this query by not grabbing the grove at all, which might be what active record is doing, but I never trust it. `@focus_areas = FocusArea.where(grove_id: @student.grove_id)`